pages/common/ssh.md

```diff
@@ -30,3 +30,7 @@
 - SSH enable agent forward:
 
 `ssh -A {{username}}@{{remote_host}}`
+
+- SSH enable X11 forwarding:
```

Comment:
Thanks! Care to expand the description slightly? Perhaps mention a usage example.

Also, this page is getting rather long. Do you think any of the current examples could be removed or combined?